docker/chatbot/Dockerfile

```diff
@@ -1,19 +1,26 @@
 # Dockerfile builds an image with a chatbot (Tino) for Tinode.
 
-FROM python:3.7-alpine
+FROM python:3.7-slim
 
 ARG VERSION=0.16
+ARG LOGIN_AS=
+ARG TINODE_HOST=tinode-srv:6061
 ENV VERSION=$VERSION
+ARG BINVERS=$VERSION
 
 LABEL maintainer="Tinode Team <info@tinode.co>"
 LABEL name="TinodeChatbot"
 LABEL version=$VERSION
 
 RUN mkdir -p /usr/src/bot
+
 WORKDIR /usr/src/bot
 
+# Volume with login cookie. Not created automatically.
+# VOLUME /botdata
```

Comment:
What is this for?

Reply:
It's a reminder which volumes have special meaning and can be mapped to external directories. I can just leave a comment without the VOLUME part but it seems clearer this way.